site/src/api/queries/workspaces.ts

```diff
@@ -0,0 +1,14 @@
+import * as API from "api/api";
+import { type WorkspacesResponse } from "api/typesGenerated";
+import { type QueryOptions } from "react-query";
+
+export function workspacesByQueryKey(query: string) {
+  return ["workspaces", query] as const;
+}
+
+export function workspacesByQuery(query: string) {
```

Comment:
I think we've just been re-using the `API` function name for these, so this one would be `workspaces`